pytest_mpl/plugin.py

```diff
@@ -364,6 +350,26 @@ def get_baseline_directory(self, item):
 
         return baseline_dir
 
+    def _download_file(self, baseline, filename):
+        # Note that baseline can be a comma-separated list of URLs that we can
+        # then treat as mirrors
+        for base_url in baseline.split(','):
+            try:
+                u = urlopen(base_url + filename)
+                content = u.read()
+            except Exception as e:
+                self.logger.debug(f'Downloading {base_url + filename} failed: {repr(e)}')
```

Comment:
I need to check this branch out and test this but:

What is the difference in output if we log at INFO vs DEBUG?

Reply:
INFO is probably more visible and might mess with tests that do `caplog`.

Reply:
I had a look at this code. By default pytest will not show any log messages for a test that passes. So currently the debug (or info) log entry is not shown unless the test also fails. However, if you run pytest with [`--log-cli-level debug`](https://docs.pytest.org/en/6.2.x/logging.html#live-logs) the debug log entries will be shown for passing tests, although this also shows a huge number of matplotlib debug messages about fonts etc. The failed download message is would probably be better as an info message anyway.

How about letting the user decide if they want a warning or an error? If the URL begins with `?`, e.g. `--mpl-baseline-path=?https://example1.com,?https://example2.com`, an info messaged is logged, and otherwise it issues a warning so no change in current behaviour?